packages/launchpad/src/setup/OpenBrowser.vue

```diff
@@ -9,7 +9,7 @@
     />
     <OpenBrowserList
       variant=""
-      :was-browser-set-in-c-l-i="query.data.value.localSettings.preferences.wasBrowserSetInCLI"
```

Comment:
yes this is a linting rule, it would be good to follow the convention of kebab-cased attributes for prop names. Vue magically provides them as camelCase inside the child component, so thing else should have to change.